.tool-versions

```diff
@@ -1,8 +1,8 @@
 # These are used for the dev environment.
 # This should match the versions used in the built product.
 nodejs 18.16.0
-elixir 1.14.4-otp-25
-erlang 25.3.2
+elixir 1.15.0-otp-26
+erlang 26.0.1
```

Comment:
We need to wait for 26.0.2 which should be released early next week, otherwise, elixir-ls doesn't work.